ui/src/shared/components/AddResourceDropdown.tsx

```diff
@@ -34,14 +35,30 @@ export default class AddResourceDropdown extends PureComponent<Props> {
   private get optionItems(): JSX.Element[] {
     const importOption = this.importOption
     const newOption = this.newOption
-    return [
+    const templateOption = this.templateOption
+
+    const items = [
       <Dropdown.Item id={newOption} key={newOption} value={newOption}>
         {newOption}
       </Dropdown.Item>,
       <Dropdown.Item id={importOption} key={importOption} value={importOption}>
         {importOption}
       </Dropdown.Item>,
     ]
+
+    if (!!this.props.onSelectTemplate) {
```

Comment:
seems odd that the existence of a handler is determining whether to render this.

Reply:
would you recommend a boolean prop instead?

Reply:
yeah that seems more informative